From 882a598c5ce4ce235ecb23f22afc2141c92910f5 Mon Sep 17 00:00:00 2001
From: Christian Hesse <mail@eworm.de>
Date: Sat, 15 Oct 2022 22:38:42 +0200
Subject: [PATCH 1/1] Fallback to `uname -n` on missing `hostname` in radtest

This should work without inetutils being installed, so add a fallback.

Signed-off-by: Christian Hesse <mail@eworm.de>
---
 src/main/radtest.in | 2 +-
 1 file changed, 1 insertion(+), 1 deletion(-)

diff --git a/src/main/radtest.in b/src/main/radtest.in
index 11e929147a..e15f92ed9b 100644
--- a/src/main/radtest.in
+++ b/src/main/radtest.in
@@ -100,7 +100,7 @@ if [ "$7" ]
 then
 	nas=$7
 else
-	nas=`hostname`
+	nas=`(hostname || uname -n) 2>/dev/null | sed 1q`
 fi
 
 (
